### DISCUSSION TITLE
Fresh install throwing webpack warnings when using nested (S)CSS

### DISCUSSION THREAD

**Discussion by janpersiel** [statamic/cms#5197] [Q&A] [answered]
So I have just installed a fresh Statamic on my Mac. The problem: As soon as I start nesting any kind of CSS I get this warning
```
1 WARNING in child compilations (Use 'stats.children: true' resp. '--stats-children' for more details)
```
In addition to that my MacOS (Monterey, 12.1) notification from Laravel Mix which usually confirms the save/compilation sends 2 similar notifications consecutively after every save. See below.
![image](https://user-images.githubusercontent.com/6762951/152499863-7b16caca-007d-4e45-be62-54c9ebd4cfbd.png)
 
 Has anyone experienced this before and knows a solution?
 - Laravel Mix v6.0.41
 - Statamic 3.2.32
 
 Another older install on the same machine updated to the same Statamic version is doing fine.
 Thanks for your help 🙏

**Comment by duncanmcclean** [comment, marked as answer]
This is more of a Laravel Mix issue than Statamic - you'd be better opening an issue or looking for issues on the Laravel Mix repository (eg. laravel-mix/laravel-mix#3196). 

Statamic just ships with Mix out of the box.

**Reply by janpersiel** [reply]
Thanks @duncanmcclean for the blazingly fast reply. I will investigate it over there. Just started here as I thought it might be relevant as it impacts the Statamic experience.

**Reply by janpersiel** [reply]
So I could trace it back to a missing(?) line of code which does not seem to be required in the other install.
The webpack.mix.js needs an additional line and it seems to work after adding it. See
https://github.com/laravel-mix/laravel-mix/issues/3196#issuecomment-1029830114

**Reply by jackmcdade** [reply]
Fixed and tagged a new release 👍
